tests/analyzers/hashers/entropy.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Tests for the Entropy hasher."""
+
+from __future__ import unicode_literals
+
+import unittest
+
+from plaso.analyzers.hashers import entropy
+
+from tests import test_lib as shared_test_lib
+from tests.analyzers.hashers import test_lib
+
+
+class EntropyTest(test_lib.HasherTestCase):
```

Comment:
+ Hasher ? since you're testing EntropyHasher

Reply:
Done